templates/next/components/foo/Form.tsx

```diff
@@ -27,6 +28,12 @@ export const Form: FunctionComponent<Props> = ({ {{{lc}}} }) => {
 
 	return (
 		<div>
+        <div>
+        <Head>
```

Comment:
Maybe be can you use "Edit ..." in both, as in the h1?

This block must also be re-indented.